src/NHSD.BuyingCatalogue.Identity.UserDatabase/Deployment/CreateExecutiveAgencyUser.sql

```diff
@@ -0,0 +1,25 @@
+﻿IF '$(CREATE_EA_USER)' = 'True'
+AND NOT EXISTS (
+    SELECT *
+      FROM dbo.AspNetUsers
+      WHERE UserName = '$(EA_USER_EMAIL)')
+BEGIN
+    DECLARE @normalizedUserName AS nvarchar(256) = UPPER('$(EA_USER_EMAIL)');
+
+    DECLARE @executiveAgencyRoleId AS nchar(5) = 'RO116';
+    DECLARE @organisationId AS uniqueidentifier = (SELECT OrganisationId FROM dbo.Organisations WHERE PrimaryRoleId = @executiveAgencyRoleId);
+
+    INSERT INTO dbo.AspNetUsers
+    (
+        Id, UserName, NormalizedUserName, PasswordHash, 
+        FirstName, LastName, Email, NormalizedEmail, EmailConfirmed, PhoneNumber, PhoneNumberConfirmed,
+        PrimaryOrganisationId, OrganisationFunction, CatalogueAgreementSigned, [Disabled],
+        AccessFailedCount, ConcurrencyStamp, LockoutEnabled, SecurityStamp, TwoFactorEnabled
+    )
+	VALUES
+	(CAST(NEWID() AS nchar(36)), '$(EA_USER_EMAIL)', @normalizedUserName, '$(EA_USER_PASSWORD_HASH)',
```

Comment:
One suggestion here would be to introduce variables for each of the environment variables. Potential to make it easier to read by grouping them into one section & also remove the duplication of the email environment variable. Just a suggestion so don't feel you need to do this.